pkg/apis/rollouts/v1alpha1/analysis_types.go

```diff
@@ -130,6 +130,8 @@ type MetricProvider struct {
 	Web *WebMetric `json:"web,omitempty"`
 	// Wavefront specifies the wavefront metric to query
 	Wavefront *WavefrontMetric `json:"wavefront,omitempty"`
+	// NewRelic specifies the newrelic metric to query
+	NewRelic *NewRelicMetric `json:"newrelic,omitempty"`
```

Comment:
The API violation in `violation_exceptions.list` is complaining that `NewRelic` has the `R` in the field name uppercased, but the JSON tag is all lowercase.  The json tag should be `newRelic` to avoid the violation.

Reply:
Thanks I'll get that fixed up!